tools/test-generator.py

```diff
@@ -46,6 +47,8 @@ def send(conn, lat, lon, course, speed, alarm, ignition, accuracy, rpm, fuel):
         params = params + (('rpm', rpm),)
     if fuel:
         params = params + (('fuel', fuel),)
+    if rfid:
+        params = params + (('rfid', rfid),)
```

Comment:
Yes, I test almost after every changes, sorry sometime superficially. 
I tested it more carefully last time.